Makefile

```diff
@@ -1049,6 +1048,7 @@ $(TARBALL): release-only $(NODE_EXE) doc
 	$(RM) -r $(TARNAME)/tools/node_modules
 	$(RM) -r $(TARNAME)/tools/osx-*
 	$(RM) -r $(TARNAME)/tools/osx-pkg.pmdoc
+	find $(TARNAME)/deps/v8/test -mindepth 1 ! -regex '.*/test/torque\(/.*\)?' -delete
```

Comment:
There's a chance this might be too sophisticated for some of our supported platforms which is why you see the use of `| xargs $(RM)` instead of `-delete` on the lines below. I don't recall the details but the more you can simplify this the better, I don't even know if `-regex` is well enough supported across platforms. We can churn out a test release with this but this make target has been known to be relied upon by users aside from us so we may get bug reports. Now we're using Linux for our source tarballs maybe this won't matter _too_ much for now but the simpler the better.

Reply:
I'll try this out on AIX (I know this has a non-GNU `find`) later when I get in the office.

Reply:
(For reference: [AIX reference for `find`](https://www.ibm.com/support/knowledgecenter/en/ssw_aix_71/f_commands/find.html) suggests neither `-mindepth` nor `-delete`.)

Reply:
@rvagg Okay have something working now on AIX which has a much more restrictive `find` than on Linux. PTAL.